src/Microsoft.OpenApi.OData.Reader/Operation/OperationHandler.cs

```diff
@@ -186,23 +186,37 @@ protected static void AppendCustomParameters(OpenApiOperation operation, IList<C
         {
             foreach (var param in customParameters)
             {
+                string documentationUrl = null;
+                string paramDescription;
+
+                if (param.DocumentationURL != null)
+                {
+                    documentationUrl = $" Documentation URL: {param.DocumentationURL}";
+                }
+
+                // DocumentationURL value is to be appended to
+                // the parameter Description property
+                if (param.Description == null)
+                {
+                    paramDescription = documentationUrl?.Remove(0, 1);
+                }
+                else
+                {
+                    paramDescription = param.Description + documentationUrl;
+                }
```

Comment:
Changing the if statements here helps someone to read the code more quickly. It also moves the declaration closer to its assignment

```suggestion
               string paramDescription = (param.Description == null) ? documentationUrl?.Remove(0, 1) : param.Description + documentationUrl;
```